### ISSUE TITLE
odd behaviour when both peptides have same sequence

### ISSUE THREAD

**Issue by colin-combe** [Rappsilber-Laboratory/spectrum#4]
<a href="https://github.com/colin-combe"><img src="https://avatars.githubusercontent.com/u/977607?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [colin-combe](https://github.com/colin-combe)**
_Tuesday Sep 01, 2015 at 13:34 GMT_
_Originally opened as https://github.com/colin-combe/spectrum/issues/10_

----

**Comment by colin-combe** [comment]
@lars-kolbowski - possibly still issue?

**Comment by lars-kolbowski** [comment]
no idea page 404'd

**Comment by colin-combe** [comment]
it 404s when peptide sequences are the same? Sounds odd...

**Comment by lars-kolbowski** [comment]
you link 404'd so I have no idea what this odd behaviour is supposed to be
https://github.com/colin-combe/spectrum/issues/10

**Comment by lars-kolbowski** [comment]
seems like you deleted the issues page on your repo so all links are dead.
See https://github.com/Rappsilber-Laboratory/xiSPEC_spectrumViewer/issues/3 and https://github.com/Rappsilber-Laboratory/xiSPEC_spectrumViewer/issues/9

**Comment by colin-combe** [comment]
There was a mix up happening because some part of the code used the peptide sequence to identify the peptide. It meant the annotation of fragments went funny, can't remember how exactly. 

Its from more than two years ago so it might be fixed.

I would just try it with two identical sequence peptides if you want to see if its still a problem.

**Comment by colin-combe** [comment]
> seems like you deleted the issues page on your repo so all links are dead.

yes - i thought having issue tracking on my version of spectrum viewer wasn't going to help us

I think we're talking about issues from before you started working on xiSPEC...
